### DISCUSSION TITLE
Engine Start Deletes Flight Plan

### DISCUSSION THREAD

**Discussion by cajunjosh** [flybywiresim/aircraft#2358] [Help]
Whether I start off using ground power or use the batteries to start the APU as soon as I start an engine with APU power established the flight plan is erased.  I can re-enter the flight plan and as soon as I start the other engine the flight plan gets erased again.  This only happens when I'm manually entering my own flight plan.  If I use a system generated plan from the main menu it works fine.

**Comment by MapleNAV-TCASFAULT** [comment]
What version of the mod do you use?

**Reply by cajunjosh** [reply]
Latest dev version with no other mods installed.
